### PR TITLE
LPS-142885 Limit the exception only for the text field type with the display style of singleline

### PR DESCRIPTION
**Pull request by marcelabc**

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 1]
```diff
@@ -1244,6 +1255,26 @@ else if (clazz == Boolean.class) {
...
+
+			InputStream inputStream = null;
+			String value = null;
+
+			try {
+				inputStream = clob.getAsciiStream();
+
+				value = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+			}
+			catch (IOException ioException) {
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1269]: can we make like:

`catch (IOException | SQLException exception) {
throw new SystemException(exception);
}
`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 2]
```diff
@@ -1244,6 +1255,26 @@ else if (clazz == Boolean.class) {
 
 			values.put(name, (Boolean)object);
 		}
+		else if (clazz == Clob.class) {
+			Clob clob = (Clob)object;
+
+			InputStream inputStream = null;
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1261]: can we put this inside the try clause?

File: modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java [hunk 3]
```diff
@@ -1244,6 +1255,26 @@ else if (clazz == Boolean.class) {
...
+				value = IOUtils.toString(inputStream, StandardCharsets.UTF_8);
+			}
+			catch (IOException ioException) {
+				throw new SystemException(ioException);
+			}
+			catch (SQLException sqlException) {
+				throw new SystemException(sqlException);
+			}
+
+			values.put(name, value);
```
Comment (gabrielwas) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java:1276]: can we put this inside the try clause?

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:forward

**Pull request closed without merging**